crates/core/src/subscription/module_subscription_actor.rs

```diff
@@ -126,7 +126,7 @@ impl ModuleSubscriptionActor {
             Command::AddSubscriber { sender, subscription } => self.add_subscription(sender, subscription).await?,
             Command::RemoveSubscriber { client_id } => self.remove_subscriber(client_id),
             Command::BroadcastCommitEvent { event, feedback_tx } => {
-                self.broadcast_commit_event(event).await;
+                tokio::task::block_in_place(|| self.broadcast_commit_event(event)).await;
```

Comment:
Can wait to do this until we observe some measurements? It's not apparent that this is, or will be, a bottleneck.

Reply:
The `block_in_place`? `broadcast_commit_event` evaluates an arbitrary set of queries, obviously the query executor has good performance but it still has the potential to block for an arbitrary amount of time.

Reply:
It really depends on the workload, right? My point is that we can't immediately tell that this will improve performance globally. It would be nice to have some measurements to back it up.

Reply:
It was less about the performance aspect and more about the correctness of blocking inside of async; although, it's irrelevant since it only ever gets called from within the module host actor anyway.